jobs/index.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: post
```

Comment:
hmmm - yeah I know but unfortunately the static layout of this new theme looks pretty sh**

Thats why I used the post . I changed it to static but we should think about changing the theme a bit